lib/cosmos/gui/qt.rb

```diff
@@ -404,8 +404,15 @@ def whatsThis(column = 0)
 end
 
 class Qt::TabWidget
-  def tab(index)
-    widget(index)
```

Comment:
It felt wrong to alias widget with tab so I reverted this and changed all instances where we used it (Script Runner only).